common/src/main/java/com/sequenceiq/cloudbreak/common/mappable/StorageType.java

```diff
@@ -0,0 +1,15 @@
+package com.sequenceiq.cloudbreak.common.mappable;
+
+public enum StorageType {
+
+    UNKNOWN,
+    S3,
+    EBS,
+    EFS,
+    BLOB;
```

Comment:
Not a big deal, but perhaps an incremental approach would be better: introduce new constants later when needed and actual support is introduced.